docs_source_files/content/introduction/the_selenium_project_and_tools.de.md

```diff
@@ -3,120 +3,108 @@ title: "Das Seleniumprojekt und die Tools"
-
-Just as Selenium RC made use of the tools of the trade available at the time,
-[Selenium WebDriver]({{< ref "/webdriver/_index.md" >}}) drives that tradition on by taking
-the browser interaction part to the browser vendor's home turf 
-and asking them to take responsibility of the backend, browser-facing implementations.
-Recently this work has evolved into a W3C standardisation process
-where the goal is to turn the WebDriver component in Selenium
-into the _du jour_ remote control library for user agents.
+### Selenium steuert Webbrowsers
+
+_Selenium_ , ist Vieles, aber im Kern ist eine Sammlung von Werkzeugen um Webbrowser
+zu automatisieren. Es nutzt die besten Mechanismen die verfügbar sind um
+Browsersinstanzen zu steuern und Benutzereingaben zu simulieren.
+
+Es ermöglicht Aktionen von Benutzern zu simulieren;
+Texteingaben,
+Auswahl von Optionen in Drop-Downs, aktivieren von Checkboxen und
+anklicken von Links auf Webseiten.
+Es ermöglicht auch viele andere Möglichkeiten wie zum 
+Beispiel Mausbewegungen, beliebige JavaScript Codeausführungen
+und vieles mehr.
+
+In erster Linie wird Selenium für Front-End Testen von Websiten verwendet.
+Selenium ist im Grunde eine _'user agent' Bibliothek_. 
+Es stehen Schnittstellen zur Verfügung, die es ermöglichen mit anderen
+Bibliotheken zu kommunizieren um Deinen Anforderungen gerecht zu werden.
+
+
+### Ein Interface für sämtliche Anforderungen
+
+Eine Leitsatz des Projektes ist es eine Schnittstelle zur Verfügung
+zu stellen um alle (relevanten) Browser zu unterstützen.
+Webbrowser sind sehr komplexe und hoch entwickelte Applikationen.
+Das Ausführen von Aktionen funktioniert in jedem Browser anders, auch
+wenn der sichtbare Teil der Aktionen bei allen gleich aussieht.
+Obwohl Texte mit der gleichen Schriftart gerendert werden,
+Bilder an der gleichen Stelle dargestellt werden
+und Links auf die gleichen Ziele verweisen.
+Was im Hintergrund passiert ist so unterschiedlich wie Tag und Nacht.
+Selenium "abstrahiert" diese Unterschiede und verbirgt diese Details
+und Feinheiten von der Person die den Programmcode schreibt.
+Dadurch wird ermöglicht Programmcode zu schreiben die einen komplexen Workflow
+abbilden und dann auf Firefox, Internet Explorer, Chrome und allen anderen
+unterstützen Browsern lauffähig sind.
+
+### Werkzeuge und Support
+
+Selenium's minimalistischer Designansatz verleiht im die Vielseitigkeit
+um als Komponenten in größeren Applikationen eingesetzt zu werden.
+Weitere Werkzeuge die unter der Schirmherrschaft von Selenium stehen
+bieten Werkzeuge um [grid of browsers]({{< ref "/grid/_index.md" >}}) zu 
+erstellen.
+Somit können Tests auf unterschiedlichen Browsern und unterschiedlichen
+Betriebssystemen verteilt auf mehrere Rechner ausgeführt werden.
+
+Stell Dir vor mehrere Computer in Deinem Serverraum or Datencenter
+starten gleichzeitig ihre Browser, klicken den Link zu Deiner Webseite,
+Formulare und Tabellen; Testen Deine Applikation rund um die Uhr.
```

Comment:
can we replace 'Formulare und Tabellen -' instead of 'Formulare und Tabellen;'

Reply:
I don't see an issue here, we can merge this.

Reply:
I used this because it was the same in English. In German ";" is unusual to use it in sentences.